Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -1178,6 +1190,12 @@ theorem extChartAt_image_nhd_mem_nhds_of_boundaryless [I.Boundaryless]
   rw [extChartAt]
   exact extend_image_nhd_mem_nhds_of_boundaryless _ (mem_chart_source H x) hx
 
```

Comment:
Very minor point: now you're putting the "interior" result after the boundaryless one; it was the other way around for the other two lemmas you inserted.